### PR TITLE
Add GET /chains/:chainId/contracts/:contractAddress 

### LINKED ISSUE #100: Add contract/<address> route
- Add `/v1/chains/<chain_id>/contracts/<contract_address>` route
- Caching support should be added
- Tests should be added
- OpenApi specification should be added

Reference: https://github.com/safe-global/safe-client-gateway/blob/89bd81400b6356d2c34804fe5f097fc983d7f1d0/src/routes/contracts/routes.rs#L67-L90

### PR DESCRIPTION
**Pull request by hectorgomezv**
This PR implements the GET contract endpoint.

- [X] Add /v1/chains/<chain_id>/contracts/<contract_address> route
- [X] Caching support should be added
- [X] Tests should be added
- [X] OpenApi specification should be added

A validator is used, following the strategy from #85 

Closes #100

### CODE REVIEW

File: src/domain/balances/balances.validator.spec.ts [hunk 1]
```diff
@@ -24,6 +22,10 @@ describe('Balances validator', () => {
 
   it('should return the data when validation succeed', () => {
     const balances = [balanceFactory(), balanceFactory()];
+    const expectedErrMessage = 'testErrMessage';
+    mockValidationErrorFactory.from.mockReturnValueOnce(
+      new HttpException(expectedErrMessage, 500),
+    );
```
Comment (moisses89) [inline src/domain/balances/balances.validator.spec.ts:28]: I din`t get this, the return value should be 500 but validator succeed?

File: src/domain/contracts/entities/contract.entity.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export class Contract {
+  address: string;
+  name: string;
+  displayName: string;
+  logoUri?: string;
+  contractAbi?: object;
+  trustedForDelegateCall: boolean;
+}
```
Comment (moisses89) [inline src/domain/contracts/entities/contract.entity.ts:8]: Following the tx-service model https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/contracts/models.py#L160 I think logo shouldn't be optional. 
I mean if no logo, the service should return empty string not `null`. 
What do you think?

File: src/routes/contracts/contracts.module.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { ContractsController } from './contracts.controller';
+import { ContractsService } from './contracts.service';
+
+@Module({
+  controllers: [ContractsController],
+  providers: [ContractsService],
+})
+export class ContractsModule {}
```
Comment (moisses89) [inline src/routes/contracts/contracts.module.ts:9]: Why modules here? :thinking:

**Review by moisses89 — CHANGES_REQUESTED** [review]
Looks good :) only some comments that only I would be ensure that you aware.

File: src/domain/contracts/entities/contract.entity.ts (hunk 2 above)
Comment (hectorgomezv) [reply src/domain/contracts/entities/contract.entity.ts:8]: Changed [here.](https://github.com/5afe/safe-client-gateway-nest/commit/7e1b1aa3dca4b4723a3768cb0241b527117a25a3)

Thanks!

**Review by moisses89 — APPROVED** [review]
:rocket:

**Pull request merged**